src/XAD/XAD.hpp

```diff
@@ -38,6 +38,11 @@
 #include <XAD/Interface.hpp>
 #include <XAD/Literals.hpp>
 #include <XAD/MathFunctions.hpp>
+#if defined(__APPLE__) && defined(__clang__)
+#if defined(__apple_build_version__) && (__apple_build_version__ >= 15000000)
+#include <XAD/StdCompatibility.hpp>
```

Comment:
We'd rather avoid this, as we specifically provide this header for users that would like to use XAD as a drop-in replacement in `std` namespace. Which errors were fixed by this - is there any chance to only include what's strictly needed to make the code work?

Reply:
This was a fix suggested to me by someone at XCelerit. Afaik it doesn't solve any errors. I'll double check and remove this if yes, follow up if no.